### PR TITLE
⚙️ Membership notifications settings

### PR DESCRIPTION
**Pull request by thesan**
- #4244

[Preferences: implementation <-> design](https://github.com/Joystream/orion/issues/126#issue-1699970710)

### CODE REVIEW

**Comment by thesan** [comment]
Maybe I should remove this commit: [Fix the svgs:icons-import script](https://github.com/Joystream/atlas/pull/4451/commits/6e056bf02be5ddcae121135b3eb1cb3619dacc3e)

I started following the doc to import icons from Figma and when I got stuck I started messing with `packages/atlas/scripts/figma-import/files.js` thinking maybe my config was wrong. Eventually I found a way too make it work with [the up to date icon page](https://www.figma.com/file/2tlBY1JQtRMoyjmjJQ9jam/Icons?type=design&node-id=304-2547&mode=design&t=IwryZh0sxuL9TkbL-0) but it's not as convenient as it use to be since you would have to reconfigure it for each icon category.

File: packages/atlas/src/views/viewer/MembershipSettingsView/MembershipNotifications/MembershipNotifications.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+export const MembershipNotifications = () => {
+  const { data, isLoading } = useMemberSettingsData()
+
+  const form = useForm<Record<string, boolean>>()
+  const {
+    reset,
+    formState: { isDirty },
+  } = form
+
+  const [isSubmitting, setIsSubmitting] = useState(false)
```
Comment (WRadoslaw) [inline packages/atlas/src/views/viewer/MembershipSettingsView/MembershipNotifications/MembershipNotifications.tsx]: WDYT about creating a variable based on `isSubmitting || isLoading` and using it to disable both the table and action bar?

File: packages/atlas/src/views/viewer/MembershipSettingsView/MembershipNotifications/MembershipNotifications.tsx [hunk 2]
```diff
@@ -0,0 +1,229 @@
+import { useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import useResizeObserver from 'use-resize-observer'
+
+import { NotificationsTable } from '@/components/NotificationsTable'
+import { EntitySettingTemplate } from '@/components/_templates/EntitySettingTemplate'
+
+import { StyledActionBar, Wrapper } from './MembershipNotifications.styles'
+
+const TABLE_STRUCTURE = [
```
Comment (WRadoslaw) [inline packages/atlas/src/views/viewer/MembershipSettingsView/MembershipNotifications/MembershipNotifications.tsx]: I'd say we can export this to `{componentName}.utils.ts`

**Review by WRadoslaw — DISMISSED** [review]
Few minor changes, plus I think we should stay with the previous icon import script

**Comment by WRadoslaw** [comment]
One more thing https://github.com/Joystream/atlas/pull/4455 take a look at this PR. Basically I needed to rearrange the layout a little bit, so the tabs on the top are always visible

**Comment by thesan** [comment]
@WRadoslaw thanks for the review.
I'll address the CR and conflicts along with the replacing the preferences key with the actual ones from Orion.

**Review by attemka — APPROVED** [review]

**Pull request merged**